tests/ops/api/v1/endpoints/test_connection_config_endpoints.py

```diff
@@ -96,6 +96,7 @@ def test_patch_http_connection(
         response = api_client.patch(url, headers=auth_header, json=payload)
         assert 200 == response.status_code
         body = json.loads(response.text)
+        print(body)
```

Comment:
this may be left over from local dev? or is it intentional?

Reply:
Thanks, I was debugging something and forgot to remove it.